rxtools/source/features/TitleKeyDecrypt.c

```diff
@@ -112,7 +112,7 @@ int GetTitleKey(unsigned char* TitleKey, unsigned int low, unsigned int high){
     unsigned int tid_high = ((high>>24)&0xff) | ((high<<8)&0xff0000) | ((high>>8)&0xff00) | ((high<<24)&0xff000000);
     unsigned int tick_size = 0x200;     //Chunk size
     if(FileOpen(&tick, "1:dbs/ticket.db", 0)){
-        unsigned char* buf = 0x22000000;
+        unsigned char* buf = TITLES;
```

Comment:
Even myself didn't see that yesterday. Using macro to expand its cast is quite good.